lib/irb/ext/tracer.rb

```diff
@@ -42,37 +32,7 @@ class Context
     #
     # See +lib/tracer.rb+ for more information.
     def use_tracer=(opt)
-      if opt
-        Tracer.set_get_line_procs(@irb_path) {
-          |line_no, *rests|
-          @io.line(line_no)
-        }
-      elsif !opt && @use_tracer
-        Tracer.off
-      end
```

Comment:
Same here, `set_get_line_procs`, `off` / `on` are all method of the v0.1.x API, without an equivalent in v0.2.x